src/components/components/schedule/Event.js

```diff
@@ -17,8 +17,12 @@ class Event extends React.Component {
                 <div className="day-img">
                 </div>
                 <div className="day-info">
-                    <h4>{this.props.item.time}</h4>
-                    <h2>{this.props.item.title}</h2>
+                    <div className="timeContainer">
+                        <h4>{this.props.item.time}</h4>
+                    </div>
+                    <div>
+                        <h2>{this.props.item.title}</h2>
+                    </div>
```

Comment:
Reverse the order of time and title tags because using flex display for `.day-schedule` will help format this row. Then use `justify-content` for `space-between` will help format both texts on either end - you won't need div tags for each of them and simply the header tags are good. You can either edit the css within `.day-schedule`, or add bootstrap classnames like `d-flex` and `justify-content-between`. 

The padding between each event is a big big, but you can adjust to something like `pb-3` on line 16.